.tekton/main-pull-request.yaml

```
@@ -299,6 +299,22 @@ spec:
           image: registry.access.redhat.com/ubi8/ubi:latest
           script: exec "$(workspaces.source.path)/source/scripts/konflux/subscription-manager/subscription-manager-bro.sh" smuggle
 
+    - name: fetch-external-networks
+      runAfter:
+      - clone-repository
+      taskSpec:
+        steps:
+          - name: fetch-external-networks
+            image: registry.access.redhat.com/ubi8-minimal:latest
+            script: |
+              #!/usr/bin/env bash
```

Please add the usual mantra for failing-fast:
```suggestion
              #!/usr/bin/env bash
              set -euo pipefail
```
Asking because the script now is more than one-liner :-)